package.json

```diff
@@ -66,7 +66,7 @@
     "d3-force": "^1.0.6",
     "d3-hierarchy": "^1.1.8",
     "d3-interpolate": "1",
-    "d3-sankey-circular": "0.33.0",
+    "d3-sankey-circular": "git@github.com:antoinerg/d3-sankey-circular.git#8943bbd049c4651dbb230aa0e55ee8aad5d85afd",
```

Comment:
https://github.com/tomshanley/d3-sankey-circular/compare/master...antoinerg:8943bbd049c4651dbb230aa0e55ee8aad5d85afd

Reply:
You might want to make a PR to `d3-sankey-circular`. Maybe the author will an opinion and/or tips about your patch

Reply:
Ok, I sent a PR upstream: https://github.com/tomshanley/d3-sankey-circular/pull/36 !

Reply:
Hmm. That's frustrating. No news from the `d3-sankey-circular` maintainers. We might have to make a new package out of your fork e.g. `@plotly/d3-sankey-circular`

Reply:
No problem! I will make a release today

Reply:
Done in 5fed1a5

Reply:
:tada: https://www.npmjs.com/package/@plotly/d3-sankey-circular